doc/config.md

```diff
@@ -46,41 +46,61 @@ you can simply use the defaults which creates an in-memory database with ID `"de
 ```
 
 At the moment we support two different backends from within Datahike: [in-memory](#in-memory) and [file-based](#file-based).
-[LevelDB](#leveldb) and [PostgreSQL](#postgresql) is supported via external libraries: [datahike-postgres](https://github.com/replikativ/datahike-postgres/) and [datahike-leveldb](https://github.com/replikativ/datahike-leveldb)
+Additionally, [JDBC](https://docs.oracle.com/javase/8/docs/technotes/guides/jdbc/) databases like [PostgreSQL](#postgresql) are supported via an external library: [datahike-jdbc](https://github.com/replikativ/datahike-jdbc/).
 
 ### in-memory
 
 - `<backend>`: `mem`
 - `id`: ID of the database
-- example: `{:store {backend :mem :id "mem-example"}}`
-- uri example (deprecated): `datahike:mem://mem-example`
+- example: 
+```clojure 
+  {:store {:backend :mem 
+           :id "mem-example"}}
+```
+- via environment variables:
+```bash
+DATAHIKE_STORE_BACKEND=mem
+DATAHIKE_STORE_CONFIG='{:id "mem-example"}'
+```
 
 ### file-based
 
 - `<backend>`: `file`
 - `path`: absolute path to the storage folder
-- example: `{:store {:backend :file :path "/tmp/file-example"}}`
-- uri example (deprecated): `datahike:file:///tmp/file-example`
+- example: 
+```clojure 
+  {:store {:backend :file 
+           :path "/tmp/file-example"}}
+```
+- via environment variables:
+```bash
+DATAHIKE_STORE_BACKEND=file
+DATAHIKE_STORE_CONFIG='{:path "/tmp/file-example"}'
+```
 
 ### Supported External Backends
 
-#### LevelDB
-
-- `<backend>`: `level`
-- `path`: absolute path to the LevelDB instance
-- example: `{:store {:backend :level :path "/tmp/level-example"}}`
-- uri example (deprecated): `datahike:level:///tmp/level-example`
-
-#### PostgreSQL
+#### JDBC
+
+- `<backend>`: `jdbc`
+- `dbtype`: [JDBC supported database](https://docs.oracle.com/cd/E19226-01/820-7688/gawms/index.html)
```

Comment:
Maybe better to reference next.jdbc: https://cljdoc.org/d/com.github.seancorfield/next.jdbc/1.2.737/doc/getting-started